pageserver/src/tenant/timeline.rs

```diff
@@ -2850,6 +2850,21 @@ impl Timeline {
         Ok(())
     }
 
+    pub(crate) async fn preserve_initdb_archive(&self) -> anyhow::Result<()> {
+        if let Some(remote_client) = &self.remote_client {
+            remote_client
+                .preserve_initdb_archive(
+                    &self.tenant_shard_id.tenant_id,
+                    &self.timeline_id,
+                    &self.cancel,
+                )
+                .await?;
+        } else {
+            tracing::info!("No remote storage configured, but was asked to backup the initdb archive for {} / {}", self.tenant_shard_id.tenant_id, self.timeline_id);
```

Comment:
Why not bail with an error here? It's only called from the mgmt API

Reply:
I don't really care either direction, I will change it to bail.

Reply:
Still planning to do that?

Reply:
Yes, will do

Reply:
Done.